packages/amplify-headless-interface/src/interface/auth/add.ts

```diff
@@ -97,6 +97,10 @@ export interface CognitoUserPoolConfiguration {
    * Account attributes that must be specified to sign up.
    */
   requiredSignupAttributes: CognitoUserProperty[];
+  /**
+   * Alias attributes that can be used for sign-up/sign-in
+   */
+  aliasAttributes?: CognitoUserAliasAttributes[];
```

Comment:
you'll need to run `yarn generateSchemas --overwrite` in the headless-interface package to generate the JSON schema updates that go along with this change